### PR TITLE
Rudimentary Go builders

### PR DESCRIPTION
**Pull request by sbinet**

### CODE REVIEW

**Comment by kszucs** [comment]
I've switched to use the official golang image, it already contains gcc and supports more architectures.

**Comment by sbinet** [comment]
thanks for the review :)

I went with downloading and unpacking the tarballs from `golang.org` because this was dramatically reducing the resulting `docker` image size.
the official ones clock at around 700Mb...

but if that's ok, then fine by me :)

File: ursabot/docker.py [hunk 1]
```diff
@@ -568,6 +568,29 @@ def conda(*packages, files=tuple()):
...
+            arch=arch,
+            os=f'ubuntu-{ubuntu_version}',
+            org='ursalab',
+            title=f'{arch.upper()} Go {go_version}',
+            steps=[
+                RUN(apt("wget", "git", "gcc")),
+                ENV(GOPATH='/go'),
+                ENV(PATH='$GOPATH/bin:/usr/local/go/bin:$PATH'),
+                ADD(docker_assets / 'install_go.sh'),
+                RUN('/install_go.sh', go_version)
```
Comment (sbinet) [inline ursabot/docker.py]: for archives' sake: this should have read:
```
RUN('/install_go.sh', arch, go_version)
```

**Comment by kszucs** [comment]
The network hasn't been a bottleneck so far, so we should prefer the official, more robust images.
We can optimize later of course. I'll deploy it soon.

**Comment by kszucs** [comment]
BTW have you tested it locally? 
https://github.com/ursa-labs/ursabot#run-a-local-instance-of-ursabot

**Comment by sbinet** [comment]
it was mostly _Gedankenexperiment_ programming before going to bed :)

**Pull request merged**